infra/templates.py

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
 #
 ################################################################################
+"""Templates for OSS-Fuzz project files."""
 
 PROJECT_YAML_TEMPLATE = """\
 homepage: "<your_project_homepage>"
+language: <programming_language>
```

Comment:
nit: give valid values as examples, as otherwise it's not clear what's expected (e.g. cpp or c++) ?